public/app/percona/integrated-alerting/components/AlertRuleTemplate/AlertRuleTemplate.service.ts

```diff
@@ -11,39 +11,44 @@ import {
   TemplatesListAPI,
 } from './AlertRuleTemplate.types';
 
-const BASE_URL = `/v1/management/alerting/Templates`;
+const BASE_URL = `/v1/alerting/templates`;
 
 export const AlertRuleTemplateService = {
   async upload(payload: UploadAlertRuleTemplatePayload, token?: CancelToken): Promise<void> {
-    return api.post(`${BASE_URL}/Create`, payload, false, token);
+    return api.post(BASE_URL, payload, false, token);
   },
   async list(payload: AlertRuleTemplateGetPayload, token?: CancelToken): Promise<TemplatesList> {
-    return api.post<TemplatesListAPI, Object>(`${BASE_URL}/List`, { ...payload, reload: true }, true, token).then(
-      ({ totals, templates = [] }): TemplatesList => ({
-        totals,
-        templates: templates.map((template) => ({
-          ...template,
-          params: template.params?.map((param) => ({
-            ...param,
-            float: param.float
-              ? {
-                  hasMin: param.float.has_min,
-                  hasDefault: param.float.has_default,
-                  hasMax: param.float.has_max,
-                  min: param.float.min,
-                  max: param.float.max,
-                  default: param.float.default,
-                }
-              : undefined,
-          })),
-        })),
+    // todo: address payload
```

Comment:
Please take a look if the payload of the api request matches what the BE expects.

Reply:
Checked. No address to add in the payload